includes/database.mysqli.inc

```diff
@@ -88,6 +88,13 @@ function db_connect($url) {
     mysqli_query($connection, 'SET NAMES utf8');
   }
 
+  // MySQL 8 has ONLY_FULL_GROUP_BY mode enabled by default. There are too many queries in Drupal 6 that do not
+  // list all columns in GROUP BY clause. So we need to turn off this mode after connecting.
+  list($version) = explode('-', mysqli_get_server_info($connection));
+  if(version_compare($version, '8.0.0', '>=')){
+    mysqli_query($connection, 'SET sql_mode=(SELECT REPLACE(@@sql_mode,\'ONLY_FULL_GROUP_BY\',\'\'));');
```

Comment:
This is only added to mysqli and not mysql. We definitely try to get everyone using PHP 7, but we want this repo to work for people who don't update too. Could this be added to mysql as well?

Reply:
Yes I thought about this. But this makes no sense since you can't connect to MySQL 8.0 with "mysql" extension due to protocol incompatibilities. So it is only possible with "mysql**i**" extension.

Reply:
Ah, ok, that makes sense!